master/buildbot/steps/master.py

```diff
@@ -213,6 +213,26 @@ def run(self):
         return defer.succeed(SUCCESS)
 
 
+class AssertRenderable(BuildStep):
```

Comment:
Would it be better to just call it Assert()

Renderable is kindof implicit in buildbot, everything could be renderable